src/app/renderer/index.tsx

```diff
@@ -294,6 +307,21 @@ export function renderGraphicalElementSVG(
         />
       );
     }
+    case "chart-container": {
+      const component = element as Graphics.ChartContainerElement;
+      return (
+        <ChartComponent
+          key={options.key}
+          chart={component.chart}
+          dataset={component.dataset}
+          width={component.width}
+          height={component.height}
+          rootElement="g"
+          sync={options ? options.chartComponentSync : false}
```

Comment:
Nitpick:  You already defaulted options to the empty object above, so this could just be `options.chartComponentSync`